mmdet/ops/nms/nms_wrapper.py

```diff
@@ -103,6 +103,21 @@ def nms(boxes, scores, iou_threshold, score_thr=0.0, max_num=-1, offset=0):
     return dets, inds
 
 
+def get_nms_from_type(nms_type: str):
+    """A function to define an NMS operation by NMS type.
+
+    Arguments:
+        nms_type (str): string with nms type (e.g. 'nms').
+
+    Returns:
+        function: NMS function or 'None' if the nms_type is not supported.
+    """
+    nms_op = None
+    if nms_type is 'nms':
```

Comment:
Please, make check `==` instead of `is`
I'm not sure that the `is` will always work.

Reply:
Done.